app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+    first_file, second_file = command_list[1], command_list[2]
```

Comment:
what about first element of the command? what if it doesn't equal 'cp'? What if command contains more then 3 words?

Reply:
> what about first element of the command? what if it doesn't equal 'cp'? What if command contains more then 3 words?

That's the question for the people who made testes for the task, but I'll fix it.

Reply:
In real life when you expect user input be ready for anything :)

Reply:
@panicua You have this as first point in your [checklist](https://github.com/mate-academy/py-copy-file/blob/master/checklist.md) 👀`Code Efficiency Make sure you check that the cp input command is correct.`